lib/api/devicestatus/index.js

```diff
@@ -54,6 +55,28 @@ function configure (app, wares, ctx) {
       });
     });
 
+    // delete record that match query
+    api.delete('/devicestatus/', ctx.authorization.isPermitted('api:devicestatus:delete'), function(req, res) {
+      var query = req.query;
+      if (!query.count) {
+        query.count = 10
```

Comment:
Shouldn't we set the default to for `count=0`, so that the user must explicitly remove records?

Reply:
@PieterGit, I've looked through some of the documentation for mongodb and it doesn't appear the count has any impact on remove().

I'll remove the count so it is not confusing.